src/components/FreelancerCard.astro

```diff
@@ -0,0 +1,141 @@
+---
+import { Image } from 'astro:assets';
+import type { ImageMetadata } from 'astro';
+const { freelancer } = Astro.props;
+
+// Import social icons
+import { Icon } from 'astro-icon/components';
+
+import SecondaryCTA from "@components/ui/buttons/SecondaryCTA.astro";
+
+// Define the type for a skill
+type Skill = string;
+
+// Define a boolean to check if any certification is true
+const hasSprykerCertifications = freelancer.sprykerCertifications.backEndDeveloper || freelancer.sprykerCertifications.solutionArchitect;
+
+// Function to convert country code to flag emoji
+function getFlagEmoji(countryCode: string) {
+  const codePoints = countryCode
+    .toUpperCase()
+    .split('')
+    .map(char => 127397 + char.charCodeAt(0));
+  return String.fromCodePoint(...codePoints);
+}
+// Import all images at build time with proper typing
+const imageFiles: Record<string, ImageMetadata> = import.meta.glob('/src/images/freelancers/*.{jpeg,jpg,png,gif}', {
+  eager: true,
+  import: 'default'
+});
+
+const forumIconFiles: Record<string, ImageMetadata> = import.meta.glob('/src/images/cq.png', {
```

Comment:
the next 6 lines look also more generic and not freelancer specific at all; please move to own component